recipe/build.sh

```diff
@@ -9,40 +9,27 @@ export CC=$(basename "$CC")
 export CXX=$(basename "$CXX")
 export FC=$(basename "$FC")
 
+EXTRA_OPTS=''
 if [ "${build_platform}" == "linux-s390x" ]; then
-  ./configure \
-      --prefix=$PREFIX \
-      --with-readline \
-      --with-libraries=$PREFIX/lib \
-      --with-includes=$PREFIX/include \
-      --with-openssl \
-      --with-gssapi  \
-      --with-icu \
-      --with-libxml \
-      --with-libxslt \
-      --with-lz4 \
-      --with-zstd \
-      --with-uuid=e2fs \
-      --with-system-tzdata=$PREFIX/share/zoneinfo \
-      PG_SYSROOT="undefined"
-else
-  ./configure \
+  EXTRA_OPTS='--with-ldap'
```

Comment:
I think this should be the other way round, you want to use ldap on all platforms except linux-s390x

Reply:
Indeed, it should be.